doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc

```diff
@@ -123,6 +123,7 @@ NOTE: Use this property for {CENTRAL} only. If you share a runtime environment w
 * `org.guvnor.m2repo.dir`: Place where the Maven repository folder is stored. Default value: `<working-directory>/repositories/kie`.
 * `org.guvnor.project.gav.check.disabled`: Disables group ID, artifact ID, and version (GAV) checks. Default value: `false`.
 * `org.kie.build.disable-project-explorer`: Disables automatic build of a selected project in Project Explorer. Default value: `false`.
+* `org.kie.builder.cache.size`: Defines the cache size of the project builder. We can configure the cache size using this property. Default value: `20`.
```

Comment:
Minor suggestion - can we remove the second sentence? It repeats the information and uses "we can" instead of "you can".

Remove "We can configure the cache size using this property."

Reply:
Ohhh! Yes! 
I was going to update it. Thank you so much. I will remove the 2nd sentence.